packages/adminrouter/extra/src/README.md

```diff
@@ -261,31 +261,188 @@ as-is/usually do not follow any of these patterns. It is possible to standardise
 it, but it does not seem to be justified. For example:
  * `util.lua` directly exposes its functions as they are stateless and shared
    between both flavours
- * `lib/auth/open.lua` uses the `.init()` pattern in order to achive different
+ * `lib/auth/open.lua` uses the `.init()` pattern in order to achieve different
    behaviour depending on the flavour.
 
-## Service Endpoints
-
-Admin Router allows Marathon tasks to define custom service UI and HTTP endpoints, which are made available via `<dcos-cluster>/service/<service-name>`. This can be achieved by setting the following Marathon task labels:
-
+## Service Endpoint
+Admin Router offers the `/service` endpoint which enables users to easily
+access some of the tasks launched on DC/OS. AR internally pre-fetches data from
+Mesos and root Marathon and stores it in the internal cache for later use when
+routing an incoming request.
+
+Services can be accessed using both application ID and by framework ID in the
+case of Mesos frameworks. For example - let's deploy a SchmetterlingDB
+framework with framework ID equal to `/group1/group2/schmetterlingA`. It will
```

Comment:
Actually I think that this should be "application ID" (inferred from later text).